core/src/no/group15/playmagic/ecs/GameMap.kt

```diff
@@ -56,6 +56,14 @@ class GameMap {
 	}
 
 
+
+	fun setEmptyTile(x: Float, y: Float) {
+		val x = toMatrixCoordX(x)
+		val y = toMatrixCoordY(y)
+		mapMatrix[y][x] = TileType.EMPTY
```

Comment:
It would be nice if this was resettable without making a new instance of the class
Maybe with a destroyed rock type that would be treated as empty, but there could then be a resetMap function that iterates over the map and sets this back to a rock type

also you might want to check that there is a rock there before setting it destroyed

Reply:
Done!